src/main/java/blusunrize/immersiveengineering/common/util/compat/computers/generic/owners/MultiblockCallbackOwner.java

```diff
@@ -27,10 +27,6 @@ public boolean canAttachTo(T candidate)
 	@Override
 	public T preprocess(T arg)
 	{
-		T master = (T)arg.master();
-		if(master!=null)
-			return master;
-		else
-			return arg;
+		return arg;
```

Comment:
This change breaks nearly every multiblock computer method, they expect to run on the master BE. I've fixed this part of the issue in the main 1.18.2 branch by passing both the master and the "direct" BE, so that computer methods can choose which one to use.

Reply:
Thanks, Malte! The way I did it was indeed a workaround, by getting the master block in the computer method itself. I will rebase and review my changes.